src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Can load a native assembly.Can load native dll
```powershell
Expected an exception, with FullyQualifiedErrorId 'EntryPointNotFoundException' to be thrown, but the FullyQualifiedErrorId was 'DllNotFoundException'. from /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1:70 char:11
    +         { [TestNativeClass2]::LoadNative() } | Should -Throw -ErrorId ?
    +           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 70
70:         { [TestNativeClass2]::LoadNative() } | Should -Throw -ErrorId "EntryPointNotFoundException"
```